src/CMakeLists.txt

```diff
@@ -1,19 +1,41 @@
 
 cmake_minimum_required(VERSION 3.1)
 
-project(Iaito VERSION 1.0.0)
+project(iaito VERSION 1.0.0)
 
-set(CMAKE_CXX_STANDARD 11)
+set(QT_MIN_VERSION "5.6.0")
 
-list(APPEND CMAKE_MODULE_PATH "${CMAKE_CURRENT_SOURCE_DIR}/cmake")
+set(CMAKE_MODULE_PATH ${CMAKE_MODULE_PATH} ${CMAKE_CURRENT_SOURCE_DIR}/cmake)
+include(CheckCXXCompilerFlag)
+include(FeatureSummary)
+include(GenerateExportHeader)
```

Comment:
Please tell me if I'm wrong, but I think CheckCXXCompilerFlag and GenerateExportHeader are unused.

Also, why would you use set instead of list(APPEND ...) for CMAKE_MODULE_PATH? And please put  ${CMAKE_CURRENT_SOURCE_DIR}/cmake in quotes.